ecs-cli/modules/cli/compose/entity/entity_helper.go

```diff
@@ -139,6 +144,10 @@ func createRegisterTaskDefinitionRequest(taskDefinition *ecs.TaskDefinition) *ec
 		request.Memory = memory
 	}
 
+	if len(tags) > 0 {
```

Comment:
[API](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_RegisterTaskDefinition.html#ECS-RegisterTaskDefinition-response-tags) docs say it takes an array, but presumably ECS handles empty array and nil the same?

Reply:
An empty slice and a nil slice pointer are essentially the same thing in Go, and the `len` of both will be 0. 

> will we get an error if we set an empty tags field?

I don't think so, but remember how once upon a time we had an issue because we were passing in an empty object instead of nil? #616